apps/consumer-client/pages/zuzalu-examples/group-proof.tsx

```diff
@@ -49,9 +52,30 @@ export default function Page() {
         .
       </p>
       <ExampleContainer>
-        <button onClick={requestZuzaluMembershipProof} disabled={valid}>
+        <button
+          onClick={() => requestZuzaluMembershipProof(serverProving)}
+          disabled={valid}
+        >
           Request Zuzalu Membership Proof
         </button>
+        <label>
+          <input
+            type="checkbox"
+            checked={serverProving}
+            onChange={() => {
+              setServerProving((checked: boolean) => !checked);
+            }}
+          />
```

Comment:
we should make this checkbox available for all the example, not just `group-proof`